internal/cli/user/profile_meta.go

```diff
@@ -0,0 +1,40 @@
+package user
```

Comment:
[question] thoughts on pushing this logic to the `local` package?

my thinking: most of our logic around `ioutil.ReadDir` and parsing files lives there already.

the potentially confusing bit: in a "naive-naming sense", this leaves us with both a `user.Profile` and a `local.Profile` type (Tim's got me on the crusade against "Meta"-like names now... 😬 )

if we _do_ go this route, I think the eventual `local.LoadProfiles` func should take in a `dir string` arg, which can be provided via the `profile.Dir()` which is available on the `user.Profile` type (which all commands are passed)

fwiw, I think what you have here is fine too... but the "meta" name got me thinking about alternatives here

Reply:
I think I agree with [May-you](https://mongodb.slack.com/archives/D022Z4CEDPC/p1652194428174429): 

> also wondering if it makes sense to actually put this into package local?
> you could then get away with the sleek-but-potentially-confusing type Profile struct (so that we're left with both a user.Profile and local.Profile)
> saying that out loud, i actually don't like it, but at least it was mentioned
> 

unless we're really striving for consistency in keeping the `ioutil.ReadDir` functionality all in the `local` package, I think it makes sense to keep the profile parsing logic together

Reply:
sold! 
I've made the following changes in the latest commit:

- moved profile_meta.go from `user` to `local` package and renamed it to environment.go
- renamed 
`ProfileMeta` to `Environment`
`user.Profiles` to `local.LoadProfileEnvs()`